packages/opds-web-client/src/components/CatalogLink.tsx

```diff
@@ -13,19 +13,19 @@ export default class CatalogLink extends React.Component<CatalogLinkProps, {}> {
   context: NavigateContext;
 
   static contextTypes: React.ValidationMap<NavigateContext> = {
-    router: PropTypes.object.isRequired,
+    router: PropTypes.object.isRequired as React.Validator<RouterType>,
     pathFor: PropTypes.func.isRequired
   };
 
   static childContextTypes: React.ValidationMap<NavigateContext> = {
-    router: PropTypes.object.isRequired
+    router: PropTypes.object.isRequired as React.Validator<RouterType>
   };
 
   // provides full router context expected by but not actually used by Link
   // see https://github.com/reactjs/react-router/blob/master/docs/API.md#contextrouter
   // and https://github.com/reactjs/react-router/blob/master/modules/PropTypes.js
   getChildContext() {
-    let noop = () => {};
+    let noop = () => { };
```

Comment:
Is the space necessary or  just something that wasn't reverted?

Reply:
Just something that wasn't reverted. I can remove it